src/DependencyInjection/SourceProviderPass.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\TranslationMigrationBundle\DependencyInjection;
+
+use Neusta\Pimcore\TranslationMigrationBundle\Source\SymfonySourceProvider;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+final class SourceProviderPass implements CompilerPassInterface
+{
+    private const SOURCE_PROVIDER_ID = SymfonySourceProvider::class;
+    private const LOADER_TAG = 'translation.loader';
```

Comment:
The one thing i do not understand: Are there any services with this tag defined? Is this a build in symfony tag?

Reply:
Yes, it comes from Symfony and allows us to support every translation file type that Symfony knows, automatically ([see here for Symfony 5.4](https://github.com/symfony/symfony/blob/3f6d50f3b872b962538468870991e9c0372dcba8/src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.php#L78-L109)).

Reply:
Jup, that's Symfony. For Example: https://github.com/symfony/symfony/blob/5.4/src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.php#L78-L109